addons/website/static/src/snippets/s_searchbar/000.xml

```diff
@@ -23,6 +23,7 @@
                 <div t-attf-class="h6 font-weight-bold #{description ? '' : 'mb-0'}" t-out="result['name']"/>
                 <p t-if="description" class="mb-0" t-out="description"/>
                 <button t-if="extra_link" class="extra_link btn btn-link btn-sm" t-att-data-target="result['extra_link_url']" t-out="extra_link"/>
+                <t t-if="parts['extra_link_html'] and widget.displayExtraLink and result['extra_link_html']" t-out="result['extra_link_html']"/>
```

Comment:
Would it actually be possible to use the old `extra_link` field ?
E.g. if `extra_link_url` is defined then use the old rendering, else consider it to be pre-rendered HTML ?

Reply:
Done in #79363